dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -11,7 +11,7 @@
 #include <zephyr/dt-bindings/gpio/gpio.h>
 #include <zephyr/dt-bindings/i2c/i2c.h>
 #include <zephyr/dt-bindings/pwm/pwm.h>
-#include <zephyr/dt-bindings/pm/imx_spc.h>
```

Comment:
How is this related to the changes in maintainers? Is this a bug you are trying to fix?

Reply:
fixing the path

Reply:
when fixing the maintainer path of include/dt-bindings/power , I noticed there is this one single file that created its own folder called include/dt-bindings/pm which is really redundant, so I am fixing this file path